sections/multicolumn.liquid

```diff
@@ -62,6 +62,9 @@
                 {%- if block.settings.text != blank -%}
                   <div class="rte">{{ block.settings.text }}</div>
                 {%- endif -%}
+                {%- if block.settings.link_label != blank -%}
+                  <a class="link animate-arrow"{% if block.settings.link != blank %} href="{{ block.settings.link }}"{% endif %}>{{ block.settings.link_label | escape }}<span class="icon-wrap">&nbsp;{% render 'icon-arrow' %}</span></a>
```

Comment:
If `block.settings.link` is `blank` we need to include `role="link"` to retain link semantics. Otherwise semantic meaning when discovered via virtual cursor is lost.

Also you could use this `role` to target and style accordingly.